### DISCUSSION TITLE
Has some way to include scss instead css ? - Warning with "*property" css hack

### DISCUSSION THREAD

**Discussion by jmtt89** [GrapesJS/grapesjs#4098] [Q&A]
I using GrapesJS in Angular Project and work well but in Angular 13 show some Warning CSS releated, all warning coming from grapesj.min.css, for some rules that include asterisk before css property.

grapesj.min.css
```
﻿.sp-container{position:absolute;top:0;left:0;display:inline-block;*display:inline;*zoom:1;z-index:9999994;overflow:hidden}
﻿...
﻿.sp-cf{*zoom:1}
﻿...
﻿.sp-replacer{margin:0;overflow:hidden;cursor:pointer;padding:4px;display:inline-block;*zoom:1;*display:inline;border:solid 1px #91765d;background:#eee;color:#333;vertical-align:middle}
﻿...
﻿.sp-palette{*width:220px;max-width:220px}
```

I know that is a hack for IE7 and before
"*property: value applies the property value in IE 7 and below. It may or may not work in future versions. Warning: this uses invalid CSS." 

I think that "compatibility hack" is generated in build phase maybe for IE7 compatibility, so if i can include scss instead of css on my Angular Project i think all that warning was removed.

Somebody knows if that is possible?

**Comment by artf** [comment]
https://github.com/artf/grapesjs/discussions/3925
